tests/test_fyle/test_tasks.py

```
@@ -490,3 +502,966 @@ def test_skip_expenses_and_post_accounting_export_summary(mocker, db):
+        "No expense groups found for changed expenses: %s in workspace %s",
+        changed_expense_ids, workspace_id
+    )
+
+
+
+
+
+
+
+
+def test_recreate_expense_groups_no_expenses_found(mocker, db):
```

Ruff (0.13.1)

Unused function argument: `db`

(ARG001)